ui/api/meshmodel.js

```diff
@@ -20,7 +19,6 @@ const CATEGORIES_ENDPOINT = '/api/meshmodels/categories';
 
 /** @type {pageOptions} */
 const defaultOptions = {
-  paginated: false,
```

Comment:
Removing paginated from here will affect where ever this defaultOptions var is used, you sure about removing this, why we need to make change here?

Reply:
I checked everywhere in the frontend, backend, and API documentation, and we are not using the `paginated` parameter anywhere.